docs/introduction/advanced-settings.mdx

```diff
@@ -309,6 +309,19 @@ Controls import path overrides during import resolution.
 
 Enables and disables resolution of imports from `sys.path`.
 
+<Note>
+For this to properly work, you may also need to enable `allow_external`.
+</Note>
+
+## Flag: `allow_external`
```

Comment:
Hm I don't think we have the performance in the current sdk to handle external resolution so this makes sense